### ISSUE TITLE
FullyQualifiedGlobalConstants reports a false positive on PHP 8.4 property hooks (get/set), and the fixer produces invalid code

### ISSUE THREAD

**Issue by caricare2** [slevomat/coding-standard#1864]
## Summary

`SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants` treats the PHP 8.4 property-hook keywords `get` and `set` as references to global constants and reports `FullyQualifiedGlobalConstants.NonFullyQualified`. When the sniff's automatic fixer runs (`phpcbf`), it prepends a backslash (`get => …` becomes `\get => …`), which is a **fatal syntax error** — so the fixer actively corrupts valid code.

## Environment

- slevomat/coding-standard: **8.31.0**
- squizlabs/php_codesniffer (PHPCSStandards): **4.0.1**
- PHP: **8.4.15**

## Ruleset

```xml
<?xml version="1.0"?>
<ruleset name="Repro">
    <rule ref="SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants"/>
</ruleset>
```

## Code to reproduce (`Example.php`)

```php
<?php

namespace App;

class Example
{
    private int $backing = 0;

    public int $value {
        get => $this->backing;
        set => $this->backing = $value;
    }
}
```

## Steps to reproduce

```bash
vendor/bin/phpcs -s --standard=ruleset.xml Example.php
```

## Actual result

```
 10 | ERROR | [x] Constant get should be referenced via a fully qualified name.
    |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants.NonFullyQualified)
 11 | ERROR | [x] Constant set should be referenced via a fully qualified name.
    |       |     (SlevomatCodingStandard.Namespaces.FullyQualifiedGlobalConstants.NonFullyQualified)
```

Running the fixer:

```bash
vendor/bin/phpcbf --standard=ruleset.xml Example.php
```

rewrites the hooks to:

```php
    public int $value {
        \get => $this->backing;
        \set => $this->backing = $value;
    }
```

which no longer parses:

```
PHP Parse error: syntax error, unexpected fully qualified name "\get", expecting identifier
```

## Expected result

No violations. `get` and `set` in a property-hook body are language keywords, not global constant references, and must not be fully qualified.

## Notes

- The same happens with a `get`-only hook (`public int $id { get => 42; }`).
- Because the sniff is auto-fixable, this is more than a noisy false positive: `phpcbf` (e.g. run in CI or a pre-commit hook) will turn passing code into a parse error.
